wgpu/src/backend/web.rs

```diff
@@ -1160,6 +1160,8 @@ impl crate::context::Context for Context {
             // Doesn't really have meaning on the web.
             present_modes: vec![wgt::PresentMode::Fifo],
             alpha_modes: vec![wgt::CompositeAlphaMode::Opaque],
+            // Statically set to RENDER_ATTACHMENT for now. See https://github.com/gfx-rs/wgpu/pull/3874
+            usages: wgt::TextureUsages::RENDER_ATTACHMENT,
```

Comment:
Currently this _must_ be RENDER_ATTACHMENT on WebGPU, so this is fine. https://gpuweb.github.io/gpuweb/#dom-gpucanvasconfiguration-usage